injector/src/main/java/edu/ucr/cs/riple/injector/changes/ChangeVisitor.java

```diff
@@ -165,6 +167,45 @@ public Modification visitClass(
     return change.computeTextModificationOn(((BodyDeclaration<?>) optionalClass.get()));
   }
 
+  @Override
+  public Modification visitLocalVariable(
+      OnLocalVariable onLocalVariable, Pair<NodeList<BodyDeclaration<?>>, ASTChange> pair) {
+    final NodeList<BodyDeclaration<?>> members = pair.a;
+    final ASTChange change = pair.b;
+    final AtomicReference<Modification> ans = new AtomicReference<>();
+    members.forEach(
```

Comment:
I personally think this would be clear with a for loop rather than `forEach`.  Then you could just early-return once the right variable is found and the change is computed.

Reply:
Sure, rewrote it with for loops f1ffe0808aca8e184ebb233863ecf67fd074881e